ngi_pipeline/utils/filesystem.py

```diff
@@ -23,7 +23,8 @@
 
 LOG = minimal_logger(__name__)
 
-def load_modules(modules_list):
+@with_ngi_config
+def load_modules(modules_list, config=None, config_file_path=None):
```

Comment:
I'm not sure about your coding conventions, but perhaps having docstrings for `config` and `config_file_path`?